src/tools/illink/src/linker/Mono.Linker.csproj

```
@@ -76,6 +76,15 @@
     <Compile Include="..\..\external\corert\**\*.cs" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <DependencyAnalysisFrameworkRoot>$(CoreClrProjectRoot)\tools\aot\ILCompiler.DependencyAnalysisFramework</DependencyAnalysisFrameworkRoot>
+    <ToolsCommonRoot>$(CoreClrProjectRoot)\tools\Common</ToolsCommonRoot>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(DependencyAnalysisFrameworkRoot)\ILCompiler.DependencyAnalysisFramework.csproj" />
```

nit: single usage can be inlined and backslash slash is included in `CoreClrProjectRoot` ([`NormalizeDirectory`](https://github.com/dotnet/runtime/blob/3f30cb9c9b392d127c0e4fbea2333dd1e6b82803/Directory.Build.props#L146) takes care of it).
```suggestion
    <ProjectReference Include="$(CoreClrProjectRoot)tools\aot\ILCompiler.DependencyAnalysisFramework\ILCompiler.DependencyAnalysisFramework.csproj" />
```